testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/metrics/base/MicroProfileMetricsBaseTestCase.java

```diff
@@ -76,4 +86,62 @@ public void testBaseMetrics() throws Exception {
         assertTrue((uptime2 - uptime1) >= sleep);
         assertTrue((uptime2 - uptime1) <= interval);
     }
+
+    @Test
+    public void testUnknownExposedSubsystem() throws Exception {
+        final ModelNode address = Operations.createAddress("subsystem", "microprofile-metrics-smallrye");
+        final ModelNode op = Operations.createWriteAttributeOperation(address, "exposed-subsystems", new ModelNode().add("bad").add("unknown"));
+        try {
+            ManagementOperations.executeOperation(managementClient.getControllerClient(), op);
+            Assert.fail("update exposed subsystems to unknown ones should fail");
+        } catch (MgmtOperationException e) {
+            ModelNode result = e.getResult();
+            assertTrue(result.get("failure-description").asString().contains("WFLYMETRICS0006"));
+        }
+    }
+
+    @Test
+    public void testExposedSubsystem() throws Exception {
+        final ModelNode address = Operations.createAddress("subsystem", "microprofile-metrics-smallrye");
+        ModelNode op = Operations.createWriteAttributeOperation(address, "exposed-subsystems", new ModelNode().add("jca").add("undertow"));
+        ModelNode result = ManagementOperations.executeOperationRaw(managementClient.getControllerClient(), op);
+        assertEquals("success", result.get("outcome").asString());
+        ServerReload.executeReloadAndWaitForCompletion(managementClient.getControllerClient(), 50000);
```

Comment:
These operations are deprecated, you should use the non-deprecated ones.

Additionally, if you really need to use a different timeout, the best option would be to use TimeoutUtil.adjust(...) wrapping your desired timeout value, which would make the test more robust for executions in slow machines.

e.g: `ServerReload.executeReloadAndWaitForCompletion(managementClient, TimeoutUtil.adjust(50000));`